packages/plugins/transaction-logger/src/index.js

```diff
@@ -285,8 +281,11 @@ export default class TransactionLogger {
         return {};
```

Comment:
The following suggestion also applies to [L294](https://github.com/embarklabs/embark/blob/1e7882684e6e38906da123d4d2d6be1ac8b8c0eb/packages/plugins/transaction-logger/src/index.js#L294).

```suggestion
         return asString ? '[]' : [];
```

Reply:
Oh, nice catch!

Reply:
Updated